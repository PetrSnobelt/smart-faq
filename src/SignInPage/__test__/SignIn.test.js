// @flow

import * as React from 'react';
import { shallow } from 'enzyme';
import SignIn from '../';

describe('SignIn', () => {
  const component = shallow(<SignIn history={{}} />);
  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
