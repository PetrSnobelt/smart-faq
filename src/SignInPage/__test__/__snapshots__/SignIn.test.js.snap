// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn should match snapshot 1`] = `
<div
  className="jsx-3069972566 SignIn"
>
  <CloseButton />
  <BackButton
    link="/"
    text="Back"
  />
  <div
    className="jsx-3069972566 picture"
  >
    <img
      alt="Help"
      className="jsx-3069972566"
      src="https://images.kiwi.com/smart-faq/woman-with-laptop@2x.jpg"
    />
  </div>
  <div
    className="jsx-3069972566 text"
  >
    <p
      className="jsx-3069972566 title"
    >
      Sign in
    </p>
    <Typography
      type="secondary"
    >
      We need your email address to send you a magic link for signing in. Then you'll be able to receive personalised help about your particular situation.
    </Typography>
  </div>
  <form
    className="jsx-3069972566"
    onSubmit={[Function]}
  >
    <label
      className="jsx-3069972566"
      htmlFor="email"
    >
      Email used for your booking:
      <div
        className="jsx-3069972566 input"
      >
        <InputText
          onChange={[Function]}
          placeholder="your@email.com"
          required={true}
          type="email"
          value=""
        />
      </div>
    </label>
    <button
      className="jsx-3069972566 get-help"
    >
      Get Help
    </button>
  </form>
  <p
    className="jsx-3069972566 or"
  >
     or 
  </p>
  <div
    className="jsx-3069972566 buttons"
  >
    <button
      className="jsx-3069972566 google"
      onClick={[Function]}
    >
      <img
        alt="Login with google"
        className="jsx-3069972566 google-icon"
        src="https://images.kiwi.com/smart-faq/google-logo.png"
      />
      <span
        className="jsx-3069972566 label"
      >
        Continue with Google 
      </span>
    </button>
    <button
      className="jsx-3069972566 facebook"
      onClick={[Function]}
    >
      <img
        alt="Login with facebook"
        className="jsx-3069972566 facebook-icon"
        src="https://images.kiwi.com/smart-faq/facebook-icon.png"
      />
      <span
        className="jsx-3069972566 label"
      >
        Continue with Facebook 
      </span>
    </button>
  </div>
  <div
    className="jsx-3069972566 kiwi-account"
  >
    <Link
      replace={false}
      style={
        Object {
          "textDecoration": "none",
        }
      }
      to="/kiwi-login"
    >
      <Typography
        type="active"
      >
        I want to use my Kiwi.com account
      </Typography>
      <img
        alt="kiwi login"
        className="jsx-3069972566 chevron"
        src="https://images.kiwi.com/smart-faq/chevron-right.png"
      />
    </Link>
  </div>
  <JSXStyle
    css=".SignIn.jsx-3069972566{width:480px;padding-top:128px;padding-bottom:50px;}div.picture.jsx-3069972566 img.jsx-3069972566{width:203px;height:156px;}div.picture.jsx-3069972566{margin:0px 127px 0px 150px;}div.text.jsx-3069972566{margin:52px 40px 32.2px 40px;}p.title.jsx-3069972566{color:#171b1e;font-size:28px;font-weight:bold;margin-bottom:8px;}label.jsx-3069972566{font-size:14px;line-height:1.43;color:#46515e;display:inline-block;width:309px;height:44px;}form.jsx-3069972566{margin:0px 40px 0px 40px;}div.input.jsx-3069972566{display:inline-block;width:100%;height:44px;margin-top:4px;}form.jsx-3069972566 button.jsx-3069972566{width:83px;height:44px;border-radius:3px;background-color:#00a991;font-size:14px;font-weight:bold;color:#ffffff;}.buttons.jsx-3069972566{margin-right:134px;margin-left:134px;}button.jsx-3069972566{width:212px;height:44px;border-radius:3px;margin-bottom:16px;border:none;cursor:pointer;display:inline-block;}button.get-help.jsx-3069972566{width:83px;background-color:#00a991;border:0;margin-left:8px;}button.google.jsx-3069972566{background-color:#d24318;}button.facebook.jsx-3069972566{background-color:#4465ad;}div.kiwi-account.jsx-3069972566{margin-top:8px;text-align:center;}img.chevron.jsx-3069972566{width:8px;height:8px;margin-left:5px;object-fit:contain;}span.label.jsx-3069972566{font-size:14px;font-weight:bold;line-height:1.43;color:#ffffff;display:block;text-align:left;}.facebook-icon.jsx-3069972566,.google-icon.jsx-3069972566{width:16px;height:16px;float:left;margin-left:13px;margin-right:13px;}p.or.jsx-3069972566{width:15px;height:23px;font-size:16px;line-height:1.4;color:#46515e;margin:0 auto;margin-top:26px;margin-bottom:24px;}@media only screen and (min-device-width:320px) and (max-device-width:480px){.SignIn.jsx-3069972566{width:100%;padding-top:22px;}div.picture.jsx-3069972566 img.jsx-3069972566{display:none;}div.text.jsx-3069972566{margin:64px 16px 33px 16px;}form.jsx-3069972566{margin:0px 16px;}label.jsx-3069972566{margin-bottom:33px;width:100%;}div.input.jsx-3069972566{width:100%;margin-right:0px;margin-bottom:8px;}form.jsx-3069972566 button.jsx-3069972566{width:100%;}.buttons.jsx-3069972566{margin-right:16px;margin-left:16px;}p.or.jsx-3069972566{margin:0 auto;margin-bottom:20px;}button.jsx-3069972566{width:100%;margin-bottom:20px;}button.get-help.jsx-3069972566{width:83px;background-color:#00a991;border:0;margin-left:0px;}@media only screen and (width:320px){.SignIn.jsx-3069972566{padding-top:0px;}}}"
    styleId="3069972566"
  />
</div>
`;
